lib/models/per-bundle-addon-cache/cache-entry.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Class for the entries in the SingletonAddonCache. The cache itself is keyed
+ * on the addon's PackageInfo.root.
+ *
+ * @class CacheEntry
+ */
+class CacheEntry {
```

Comment:
This class doesn't obviously provide value over using a properly shaped POJO. 🤔

Reply:
Chated with @davecombs about this, he is working to move it inline (will still likely use a class though)

Reply:
No longer used.